groovy/postInit/chemistry/inorganic_chemistry/elements/d_block/group10/PlatinumGroupChain.groovy

```diff
@@ -9,117 +9,876 @@ MACERATOR = recipemap('macerator')
 
 //MODERN SEPARATION PROCESSES
+GRAVITY_SEPARATOR.recipeBuilder() 
+        .inputs(ore('dustPentlandite'))
+        .outputs(metaitem('dustSiftedPentlandite'))
+        .chancedOutput(metaitem('dustUltramaficTailings'), 2500, 0)
+        .EUt(Globals.voltAmps[1])
+        .duration(40)
+        .buildAndRegister()
+
+MIXER.recipeBuilder()
+        .inputs(ore('dustSiftedPentlandite') * 8)
+        .fluidInputs(fluid('distilled_water') * 2000)
+        .fluidOutputs(fluid('impure_pentlandite_slurry') * 2000)
+        .EUt(Globals.voltAmps[3])
+        .duration(80)
+        .buildAndRegister()
+
+FF.recipeBuilder()
+        .fluidInputs(fluid('impure_pentlandite_slurry') * 2000)
+        .notConsumable(ore('dustAluminiumNitride'))
+        .notConsumable(ore('dustPotassiumOctylHydroxamate'))
+        .notConsumable(fluid('tnt_slurry') * 100)
+        .fluidOutputs(fluid('pentlandite_slurry') * 1000)
+        .fluidOutputs(fluid('ultramafic_tailing_slurry') * 1000)
+        .EUt(Globals.voltAmps[3])
+        .duration(80)
+        .buildAndRegister()
+
+CLARIFIER.recipeBuilder()
+        .fluidInputs(fluid('pentlandite_slurry') * 1000)
+        .outputs(metaitem('dustFlotatedPentlandite') * 16)
+        .fluidOutputs(fluid('wastewater') * 1000)
+        .duration(20)
+        .EUt(Globals.voltAmps[1])
+        .buildAndRegister()
+
+//The pelletized material is smelted in a shaft furnace to form a 
+//copper – nickel matte. Oxygen is then blown into the converter
+//to oxidize the iron sulfide selectively to iron oxide, 
+//which forms a slag.
+EBF.recipeBuilder()
+        .inputs(ore('dustFlotatedPentlandite') * 8)
+        .outputs(metaitem('dustGreenMatte') * 8)
+        .duration(20)
+        .blastFurnaceTemp(2700)
+        .EUt(Globals.voltAmps[2])
+        .buildAndRegister()
+
+ROASTER.recipeBuilder()
+        .inputs(ore('dustGreenMatte') * 8)
+        .fluidInputs(fluid('oxygen') * 4000)
+        .outputs(metaitem('white_matte') * 8)
+        .outputs(metaitem('dustIronIiiOxide') * 2)
+        .fluidOutputs(fluid('sulfur_dioxide') * 2000)
+        .duration(200)
+        .EUt(Globals.voltAmps[2])
+        .buildAndRegister()
+
+MACERATOR.recipeBuilder()
+        .inputs(metaitem('white_matte'))
+        .outputs(metaitem('dustWhiteMatte'))
+        .duration(100)
+        .EUt(2)
+        .buildAndRegister()
+
+//method described in https://patentimages.storage.googleapis.com/2b/70/5a/cbb5549831857c/US4571262.pdf
+
+//atmospheric leach
+BR.recipeBuilder()
+        .inputs(ore('dustWhiteMatte') * 2)
+        .fluidInputs(fluid('air') * 9750)
+        .fluidInputs(fluid('sulfuric_acid') * 1000)
+        .fluidOutputs(fluid('oxidized_pgm_leach') * 1000)
+        .duration(200)
+        .EUt(Globals.voltAmps[2])
+        .buildAndRegister()
+
+BR.recipeBuilder()
+        .inputs(ore('dustWhiteMatte'))
+        .fluidInputs(fluid('air') * 9750)
+        .fluidInputs(fluid('acidic_nickel_copper_sulfate_solution') * 1000)
```

Comment:
acidic_nickel_copper_sulfate_solution is impossible to get. Either this material needs to be removed or recipes are needed for this material.